documentation/development-docs/examples/private-link-resource-example.md

```diff
@@ -17,7 +17,11 @@ API for `Get` private link resource and private endpoint connection need to be r
 "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{provider}/{Top-Level-Resource}/{Top-Level-Resource-Name}/privateEndpointConnections"
```

Comment:
It is API for listing private endpoint connections. Get private endpoint connection API is still available even list API is not there.

Could you provide 2 examples if you think it's hard to explain it. For instance, one comes from monitor, another comes from keyvault